fjord/feedback/views.py

```diff
@@ -45,8 +46,15 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    template = 'feedback/thanks.html'
-    return render(request, template)
+    user_opinion = Response.objects.get(id=request.session['opinion_id'])
+    length = len(user_opinion.description.split())
+
+    if waffle.flag_is_active(request, 'thankyou') and user_opinion.locale == 'en-US' and user_opinion.happy == False and length >= 7:
+        template = 'feedback/thanks.html'
+        return render(request, template)
+    else:
+        template = 'feedback/thanks.html'
+        return render(request, template)
```

Comment:
Lines 54 and 57 are identical and use a variable set in lines 53 and 56. So it's better to do it like this:

```
if blah blah blah:
    template = 'feedback/thanks.html'
else:
    template = 'feedback/thanks.html'
return render(request, template)
```